src/js/core/Utils.js

```diff
@@ -829,6 +829,38 @@ var Utils = {
 
 
     /**
+     * Convert an byte array into a octal string
```

Comment:
I can see why you've put this in Utils because that's what I've done with the bin and hex operations, however I think in this particular case it would make more sense to just put this code directly into the `ByteRepr.runToOct` and `ByteRepr.runFromOct` functions. It's probably not something that will be used elsewhere in the app and if it is, we can always break it out into Utils again.